babel.config.js

```diff
@@ -15,5 +15,6 @@ module.exports = {
   plugins: [
     "@babel/plugin-transform-async-generator-functions",
     "@babel/plugin-transform-class-properties",
+    "@babel/plugin-transform-private-methods",
```

Comment:
From what I can understand the option would just add the methods starting with `#` as properties starting with underscore on the build files and the default creates a separate method (outside the class) that is invoked instead of `this.#privateMethod()`.

We do not have any restrictions or preferences on both so I rather use the default, the caution is that there is a higher level config that achieves the same result, but since we do not use this option we are not affected.